app/javascript/src/modules/geo_viewer.js

```diff
@@ -200,7 +200,13 @@ export default {
       layer = L.geoJSON(data, {color: color, opacity: opacity})
       geoJSON = true;
     }
-    this.map.setView(layer.getBounds().getCenter());
+
+    try {
```

Comment:
Would you mind adding a comment here telling the reader what we're trapping for and how an item would get into this situation?  Can you give an exemplar object so that people can test this path?

Reply:
Yup. done.